checkbox-provider-ce-oem/tox.ini

```diff
@@ -6,18 +6,27 @@ skipsdist=True
 [testenv]
 allowlist_externals = rm
 commands =
-    pip -q install -e ../../checkbox-ng
+    pip -q install ../../checkbox-ng
     # Required because this provider depends on checkbox-support parsers & scripts
-    pip -q install -e ../../checkbox-support
+    pip -q install ../../checkbox-support
     rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-resource.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-base.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-certification-client.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-certification-server.provider
+    rm -f /var/tmp/checkbox-providers-develop/plainbox-provider-docker.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-gpgpu.provider
+    rm -f /var/tmp/checkbox-providers-develop/com.canonical.certification.checkbox-provider-iiotg.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-sru.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-tpm2.provider
+    rm -f /var/tmp/checkbox-providers-develop/checkbox-provider-ce-oem.provider
     # Install all providers in develop mode to make sure everything works fine
+    {envbindir}/python3 ../resource/manage.py develop
     {envbindir}/python3 ../base/manage.py develop
     {envbindir}/python3 ../certification-client/manage.py develop
     {envbindir}/python3 ../certification-server/manage.py develop
     {envbindir}/python3 ../docker/manage.py develop
     {envbindir}/python3 ../gpgpu/manage.py develop
     {envbindir}/python3 ../iiotg/manage.py develop
-    {envbindir}/python3 ../resource/manage.py develop
     {envbindir}/python3 ../sru/manage.py develop
     {envbindir}/python3 ../tpm2/manage.py develop
     {envbindir}/python3 manage.py develop
```

Comment:
same here
please remove the command to install following provider
- plainbox-provider-docker
- checkbox-provider-gpgpu
- checkbox-provider-iiotg
- checkbox-provider-sru
- checkbox-provider-tpm

Reply:
If I remove these providers, checkbox will show the warning about some jobs couldn't be found if some test plans included. If you don't mind, I could remove these.

Reply:
![image](https://github.com/canonical/checkbox-provider-ce-oem/assets/32028620/9dca6253-be2b-4968-bee7-7e3b33f2108a)

Reply:
yes, that's expected behavior.
and for those providers, they should be validated in checkbox github action, so we could ignore them.

Reply:
I've followed checkbox repo to update pyYAML version to 6.0.1 to solve tox issue.